ssz/src/main/java/org/ethereum/beacon/util/ssz/SSZAnnotationSchemeBuilder.java

```diff
@@ -0,0 +1,176 @@
+package org.ethereum.beacon.util.ssz;
+
+import javafx.util.Pair;
```

Comment:
May be let's find some 'tuple classes' dependency instead of using javafx lib? 
It especially make sense when using Java9, since JavaFX would be huge additional dependency

Reply:
Apache Commons seems reasonable [replacement](http://commons.apache.org/proper/commons-lang/javadocs/api-3.1/org/apache/commons/lang3/tuple/Pair.html) (it has a lot of other useful features besides `Pair`)
Another option I found is https://www.javatuples.org/ , though using `Triplet, etc. ... ` already smells badly

Reply:
Added `org.javatuples` globally